### ISSUE TITLE
Account model may be missing contact relations

### ISSUE THREAD

**Issue by jochakovsky** [sportngin/active_zuora#72]
Seems like instances of ActiveZuora::Account do not have relations to their bill-to and sold-to contacts.

The following works:

```
ActiveZuora::Account.find('valid_account_id').contacts
ActiveZuora::Account.find('valid_account_id').bill_to_id
ActiveZuora::Account.find('valid_account_id').sold_to_id
```

But the following relations seem to be missing:

```
ActiveZuora::Account.find('valid_account_id').bill_to
ActiveZuora::Account.find('valid_account_id').sold_to
```

Thanks!

**Comment by anfleene** [comment]
Thanks for submitting an issue! The problem is likely with this code https://github.com/sportngin/active_zuora/blob/master/lib/active_zuora/generator.rb#L124-L134

My guess is that it's checking `if field? :bill_to` when it should be checking `if field? :bill_to_id`

Want to take a shot at updating those? Should be pretty easy to verify once changed.

Thanks!

**Comment by anfleene** [comment]
Fixed in [AZ  2.5.1](https://rubygems.org/gems/active_zuora/versions/2.5.1)
